src/pages/workspaces/workspace/analysis/AzureComputeModal.js

```diff
@@ -0,0 +1,324 @@
+  )(async () => {
+    //TODO: metrics onclick
+    //sendCloudEnvironmentMetrics()
+
+    //each branch of the cond should return a promise
+    await Utils.cond(
+      [viewMode === 'deleteEnvironment',
+        () => Ajax().Runtimes.runtimeV2(workspaceId, currentRuntimeDetails.runtimeName).delete()], //delete runtime
+      [Utils.DEFAULT, () => {
+        const disk = {
+          size: computeConfig.diskSize,
+          //We do not currently support re-attaching azure disks
+          name: Utils.generatePersistentDiskName(),
+          labels: { saturnWorkspaceNamespace: namespace, saturnWorkspaceName: workspaceName }
+        }
+
+        return Ajax().Runtimes.runtimeV2(workspaceId, Utils.generateRuntimeName()).create({
+          region: computeConfig.region,
+          machineSize: computeConfig.machineType,
+          labels: {
+            saturnWorkspaceNamespace: namespace,
+            saturnWorkspaceName: workspaceName
+          },
+          disk
+        })
+      }]
+    )
+
+    onSuccess()
+  })
+
+  const renderMainForm = () => {
+    return h(Fragment, [
+      div({ style: { padding: '1.5rem', borderBottom: `1px solid ${colors.dark(0.4)}` } }, [
+        renderTitleAndTagline(),
+        renderCostBreakdown()
+      ]),
+      div({ style: { padding: '1.5rem', overflowY: 'auto', flex: 'auto' } }, [
+        renderApplicationConfigurationSection(),
+        renderComputeProfileSection(),
+        renderBottomButtons()
+      ])
+    ])
+  }
+
+  //TODO this does not actually compute cost as is, see IA-3348
+  //It is possible that once we compute the cost, we would like to parameterize this and make it a shared function between the equivalent in ComputeModal
+  const renderCostBreakdown = () => {
+    return div({
+      style: {
+        backgroundColor: colors.accent(0.2),
+        display: 'flex',
+        borderRadius: 5,
+        padding: '0.5rem 1rem',
+        marginTop: '1rem'
+      }
+    }, [
+      _.map(({ cost, label, unitLabel }) => {
+        return div({ key: label, style: { flex: 1, ...computeStyles.label } }, [
+          div({ style: { fontSize: 10 } }, [label]),
+          div({ style: { color: colors.accent(1.1), marginTop: '0.25rem' } }, [
+            span({ style: { fontSize: 20 } }, [cost]),
+            span([' ', unitLabel])
+          ])
+        ])
+      }, [
+        { label: 'Running cloud compute cost', cost: Utils.formatUSD(0), unitLabel: 'per hr' },
+        { label: 'Paused cloud compute cost', cost: Utils.formatUSD(0), unitLabel: 'per hr' },
+        {
+          label: 'Persistent disk cost',
+          cost: Utils.formatUSD(0),
+          unitLabel: 'per month'
+        }
+      ])
+    ])
+  }
+
+  const renderDeleteEnvironment = () => {
+    return div({ style: { ...computeStyles.drawerContent, ...computeStyles.warningView } }, [
+      h(TitleBar, {
+        id: titleId,
+        shouldHideCloseButton,
```

Comment:
Close button is still visible. I think the name of the variable is wrong.

![image](https://user-images.githubusercontent.com/484484/166685313-f7da855b-733d-4f3a-95ad-5fa2a3d470ba.png)